### ISSUE TITLE
[Feature]: High Quality, Non-Pixelated Profile Images

### ISSUE THREAD

**Issue by highschooldropout** [simplex-chat/simplex-chat#2911]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### Feature

Files uploaded as profile images should retain their quality instead of being degraded to pixelated blurriness. File size limit should be at least a couple hundred kilobytes, resolutions up to 500x500 depending on device. Support jpegs, pngs and gifs.

**Comment by epoberezkin** [comment]
Yes, the challenge is where/how to store them within the current design constraints. Possible options are:
1. send it on demand to every connected contact. Seems like lots of unnecessary traffic.
2. upload to XFTP - either requires introducing long term non-expiring files, which seems bad idea just for profile pictures, or some complex logic to manage expiration...

Design constraints will be changing though, so it'll happen, just probably not right now.
